server/src/main/java/org/elasticsearch/search/SearchService.java

```diff
@@ -1096,19 +1096,14 @@ public IndicesService getIndicesService() {
         private boolean canMatch;
 
         public CanMatchResponse(StreamInput in) throws IOException {
+            super(in);
             this.canMatch = in.readBoolean();
```

Comment:
nit: I guess it doesn't make much difference since a primitive boolean is already kind of unmutable but it could be final now?

Reply:
oh yea I forgot that I think